app/models/user.rb

```diff
@@ -337,4 +337,8 @@ def favorite_collections_for_project(project)
   def subjects_count_cache_key
     @subjects_count_cache_key ||= "User/#{id}/uploaded_subjects_count"
   end
+
+  def subject_count_cache_expiry
+    ENV.fetch("UPLOADED_SUBJECTS_COUNT_CACHE_EXPIRY", 1.hour)
```

Comment:
Where does this get set? Is there a corresponding infrastructure repo PR or docker image update to add this?